### ISSUE TITLE
getting TypeError: $.bridget is not a function on my live server - works locally

### ISSUE THREAD

**Issue by propertunist** [desandro/jquery-bridget#11]
i have been configuring isotope for use on my website that is based on the elgg framework (www.elgg.org). elgg uses requireJS and offers it's own functions for including AMD and non AMD JS files. i found that the default recommended method for loading isotope did not work on my site and i was about to give up when i found this issue and alternative method which does work for me: https://github.com/metafizzy/isotope/issues/1125

so i was able to get the code to run locally (along with 'imagesLoaded'). however, when i came to test my code on my remote server, i saw an unexpected error in the browser console  - the same as is listed here: https://github.com/desandro/jquery-bridget/issues/6
nothing i have changed so far has resolved this for me, including locally hosting the bridget file and calling it manually, instead of relying on the version that is included in isotope.

any tips welcome!

**Comment by propertunist** [comment]
the only other clue i have presently is that i sometimes see an error in the browser console relating to not being able to access this file : 
```
https://mysite.org/cache/1468699556/default/ev-emitter/ev-emitter.js
```
which shows that for some reason the system is attempting to load this js file from my site's own cache rather than from wherever it would usually be loaded from.

**Comment by desandro** [comment]
jQuery Bridget v2.0.0 had a RequireJS bug. It was fixed in v2.0.1. You should be able to update `jquery-bridget` and get the fix. Let me know if this does not resolve it.

Otherwise, if you're using isotope.pkgd.js, you'll have to manually edit the file, or wait until the next version of Isotope is released.

**Comment by propertunist** [comment]
oh ok, thanks for clarifying. as it turns out, i actually need to use a justified-gallery plugin to get the effect i am looking for, not a masonry plugin - so i no longer need to resolve this.
